packages/e2e-tests/src/specs/editor/publishPanel/adminUser.js

```diff
@@ -82,7 +82,7 @@ describe('Publish panel in document tab', () => {
     await page.keyboard.type('auth');
 
     // wait for search results
-    await page.waitForTimeout(200);
+    await page.waitForTimeout(1000);
```

Comment:
Need a more robust solution here vs adding more time to the timeout -- the search needs to catchup before the expectations.

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/11782#discussion_r908348908